bin/resources/local.rb

```diff
@@ -175,7 +182,6 @@ def start_local_server(project_directory, mongo_directory, ruby_path, worker_num
 
   state_file = ::File.join(project_directory, cluster_name + '_configuration.json')
   receipt_file = ::File.join(project_directory, cluster_name + '_configuration.receipt')
```

Comment:
@rHorsey can we get rid of this receipt file now?  @evanweaver says he is no longer using it